### PR TITLE
feat(FTL-3528): removed bloom-vault from Affinidi Core SDK

### PR DESCRIPTION
**Pull request by oleksiipiliugin**
Bloom-vault removed totally from SDK code

### CODE REVIEW

File: sdk/core/test/integration/index.test.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ import './env'
 import { useNodeFetch } from '@affinidi/platform-fetch-node'
 useNodeFetch()
 
-require('./migration')
-require('./services')
-require('./AffinityWallet.test')
-require('./NetworkMember.test')
-require('./otp/index.test')
+// require('./migration')
```
Comment (anton-iskryzhytskyi) [inline sdk/core/test/integration/index.test.ts:7]: please uncomment

File: common-libs/common/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/common",
-  "version": "2.3.0",
+  "version": "3.0.0",
```
Comment (anton-iskryzhytskyi) [inline common-libs/common/package.json:3]: despite this library uses libraries that have to be update with major version, this library (common) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

File: common-libs/did-auth-lib/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/affinidi-did-auth-lib",
-  "version": "2.2.0",
+  "version": "3.0.0",
```
Comment (anton-iskryzhytskyi) [inline common-libs/did-auth-lib/package.json:3]: despite this library uses libraries that have to be update with major version, this library (did-auth-lib) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

File: common-libs/user-management/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/user-management",
-  "version": "1.5.0",
+  "version": "2.0.0",
```
Comment (anton-iskryzhytskyi) [inline common-libs/user-management/package.json:3]: despite this library uses libraries that have to be update with major version, this library (user-management) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

File: common-libs/common/package.json (hunk 2 above)
Comment (oleksiipiliugin) [reply common-libs/common/package.json]: changed

File: common-libs/did-auth-lib/package.json (hunk 3 above)
Comment (oleksiipiliugin) [reply common-libs/did-auth-lib/package.json]: changed

File: common-libs/user-management/package.json (hunk 4 above)
Comment (oleksiipiliugin) [reply common-libs/user-management/package.json]: changed

File: sdk/core/test/integration/index.test.ts (hunk 1 above)
Comment (oleksiipiliugin) [reply sdk/core/test/integration/index.test.ts]: fixed

**Review by cosminm-affnd — DISMISSED** [review]

**Review by anton-iskryzhytskyi — APPROVED** [review]
LGTM

**Comment by anton-iskryzhytskyi** [comment]
one note, please for branch names give some description of what is going. In other case reviewer should go to jira to understand the goal of the changes

**Review by cosminm-affnd — APPROVED** [review]

**Pull request merged**